src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/KeyQuotedValuePairListEncoding.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+namespace Microsoft.VisualStudio.ProjectSystem.Debug
+{
+    /// <summary>
+    ///  Encoding corresponding a format of keys eith a respective value that
+    ///  is surrounded in quotes, ex. key1="value1"
+    /// </summary>
+    internal sealed class KeyQuotedValuePairListEncoding
```

Comment:
you can add a new interface, IEncoding, that KeyQuotedValuePairListEncoding and KeyValuePairListEncoding both implement, so that we might be able to eventually export these encodings and make them available

Reply:
I would hold off on doing that until we have an explicit need. Once we make an interface or implementation `public` it becomes much harder to change later.

Reply:
@tmeschter it would be internal